docs/lib/datastore/fragments/ios/auth-model.md

```diff
@@ -0,0 +1,104 @@
+<amplify-callout>
```

Comment:
We should have a section with a pre-req that API is required and that this is not supported for "offline modes" since that is a primary use case.  If we do this, your documentation below which references API may change a bit...